components/infobox/wikis/smite/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,97 @@
+---
+-- @Liquipedia
+-- wiki=smite
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Page = require('Module:Page')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector')
+local Player = Lua.import('Module:Infobox/Person')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local ROLES = {
+	-- Players
+	['solo'] = {category = 'Solo players', variable = 'Solo', isplayer = true},
+	['jungler'] = {category = 'Jungle players', variable = 'Jungler', isplayer = true},
+	['support'] = {category = 'Support players', variable = 'Support', isplayer = true},
+	['mid'] = {category = 'Mid Lane players', variable = 'Mid', isplayer = true},
+	['carry'] = {category = 'Carry players', variable = 'Carry', isplayer = true},
```

Comment:
which of those are actually needed for this wiki?
like if there are roles not applicable for it kick them
(no idea what smite is so no idea which ones are applicable and which are not)

Reply:
These are the standard 5 for the game so all these will need to be there
(newest revision now contain only the needed ones)